grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -271,4 +271,68 @@ class CycleCountService {
         }
         return cycleCounts.collect { CycleCountDto.createFromCycleCountItems(it.cycleCountItems) }
     }
+
+    CycleCountDto submitCount(CycleCountSubmitCountCommand command) {
+        command.cycleCount.cycleCountItems.each { CycleCountItem cycleCountItem ->
+            updateCycleCountItemForSubmit(cycleCountItem, command.refreshQuantityOnHand, command.failOnOutdatedQuantity)
+            determineCycleCountItemStatus(cycleCountItem, command.requireRecountOnDiscrepancy)
+        }
+        if (command.cycleCount.status == CycleCountStatus.COUNTED) {
```

Comment:
I think we want READY_TO_REVIEW here, not COUNTED. We use COUNTED to indicate that there is a discrepancy but we haven't started recounting yet.